lib/api/include/irods/ticket_administration.hpp

```diff
@@ -0,0 +1,598 @@
+#ifndef IRODS_TICKET_ADMINISTRATION_HPP
+#define IRODS_TICKET_ADMINISTRATION_HPP
+
+#undef RxComm
+
+#ifdef IRODS_TICKET_ADMINISTRATION_ENABLE_SERVER_SIDE_API
+#    define RxComm RsComm
+
+struct RsComm;
+#else
+#    define RxComm RcComm
+
+struct RcComm;
+#endif // IRODS_TICKET_ADMINISTRATION_ENABLE_SERVER_SIDE_API
+
+#include <string_view>
+#include <type_traits>
+#include <string>
+
+///
+/// \brief Namespace to hold the ticket administration features
+///
+namespace irods::experimental::administration::ticket
+{
+    ///
+    /// \brief Struct tag to indicate admin privilige
+    ///
```

Comment:
Minor: Don't need the empty triple-slashes at the beginning and ends for these.

Reply:
privilige -> privilege